cmd/otelcontribcol/go.mod

```
@@ -4,7 +4,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontrib
 
 go 1.22.0
 
-toolchain go1.22.6
+toolchain go1.22.1
```

just a good idea to not touch this, it will burn
```suggestion
toolchain go1.22.6
```

---

Not even sure how this got changed. 🚀

---

still getting failure even after running
`make genotelcontribcol` 😢 
https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/10722164772/job/29732593501?pr=34797